youtube_dl/extractor/nick.py

```diff
@@ -91,3 +91,29 @@ def _real_extract(self, url):
             {'siteKey': 'nick.de'})
 
         return self._get_videos_info_from_url(mrss_url, video_id)
+
+
+class NickNightAtIE(MTVServicesInfoExtractor):
+    IE_NAME = 'nicknight.de'
+    _VALID_URL = r'https?://(?:www\.)nicknight\.(?:de|at|tv)/(?:playlist|shows)/(?:[^/]+/)*(?P<id>[^/?#&]+)'
+    _TESTS = [{
+        'url': 'http://www.nicknight.at/shows/977-awkward/videos/85987-nimmer-beste-freunde',
+        'only_matching': True,
+    }, {
+        'url': 'http://www.nicknight.at/shows/977-awkward',
+        'only_matching': True,
+    }, {
+        'url': 'http://www.nicknight.at/shows/1900-faking-it',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id)
+
+        mrss_url = update_url_query(self._search_regex(
+            r'mrss: (["\'])(?P<url>http.+?)\1', webpage, 'mrss url', group='url'),
+            {'siteKey': 'nicknight.de'})
```

Comment:
I can't find info on how to look what the web player sends in the browser. Is there an addon I need to install on chrome or Firefox? Or do I have to use wireshark and search the tcp-dump?

I've looked through the downloaded files with curl and I removed the update_url_query call